app/javascript/src/components/Dashboard/Contacts/ContactList/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from "react";
+
+import EmptyContactsListImage from "images/EmptyNotesList";
+import { Button, PageLoader } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import contactApi from "apis/contact";
+import EmptyState from "components/Common/EmptyState";
+
+import NewContactPane from "../Pane/Create";
+import Table from "../Table";
+
+const Contacts = () => {
```

Comment:
- What is the advantage of keeping the component name as Contacts and the filename as `ContactList/index.jsx`? The question boils down to why you are not naming this component as `ContactList`. If you had named the namespaces as `Contacts`, then I wouldn't have raised this question in the first place.
- Also what is the advantage of again prefixing with the `Contact` keyword when it's already within the `Contacts` namespace? `Dashboard/Contacts/ContactList/index.jsx`

Read:
- https://academy.bigbinary.com/learn-react/pages/structuring-code
- https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react

Reply:
I have rectified this mistake. Thank you for pointing this out.